e2e/provision/playbooks/roles/install/tasks/main.yml

```diff
@@ -15,6 +15,7 @@
     - {pkg: porch-dev, version: porch-dev/v2, namespaces: [porch-fn-system, porch-system]}
     - {pkg: nephio-controllers, version: nephio-controllers/v5, namespaces: [nephio-system]}
     - {pkg: configsync, version: configsync/v1, namespaces: [config-management-monitoring, config-management-system, resource-group-system]}
+    - {pkg: network-config, version: network-config/v1, namespaces: []}
```

Comment:
This change will require the addition of a system test [here](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/install/molecule/default/tests/test_default.py)

Reply:
Done, thanks!